src/shared_modules/content_manager/src/actionOrchestrator.hpp

```diff
@@ -79,25 +97,32 @@ class ActionOrchestrator final
     /**
      * @brief Run the content updater orchestration.
      *
-     * @param offset Offset value from the on-demand request. If equals zero and \p type is CONTENT, the current
-     * offset will be reset to zero. If \p type is OFFSET, this value will be used to perform the offset update.
-     * @param type Type of update the orchestrator should perform.
+     * @param updateData Update orchestration data.
+     *
+     * @note If the \p updateData offset equals zero and the \p updateData type is CONTENT, the current offset will be
+     * reset to zero. If \p updateData type is OFFSET, this value will be used to perform the offset update.
```

Comment:
I think this kind of documentation would fit better in the `UpdateData` definition

Reply:
Moved.